Source/Editor/Surface/Archetypes/Material.cs

```diff
@@ -281,7 +281,7 @@ internal enum MaterialTemplateInputsMapping
                 Title = "Material",
                 Description = "Main material node",
                 Flags = NodeFlags.MaterialGraph | NodeFlags.NoRemove | NodeFlags.NoSpawnViaGUI | NodeFlags.NoSpawnViaPaste | NodeFlags.NoCloseButton,
-                Size = new Float2(150, 300),
+                Size = new Float2(180, 300),
```

Comment:
I know this is being increased for increased font reasons. You could make it more dynamic doing something like this: Style.Current.FontSmall.MeasureText("Tessellation Multiplier").X + 30. As then it should size with the font.

Reply:
In that case, I would like to make every one of the nodes more dynamic. Maybe there is a way to calculate the minimum size horizontally? Like `max(HeaderText.Measure, (horizontal_port_count * (PortSize + PortPadding) + longest_text_left.Measure + longest_text_right.Measure + text_padding*2))` but it would be more complicated than that..

Reply:
It may be good to standardize on some equation like that though. Might have to play with it a little to get it right.

Reply:
I added auto-resizing.